addons/point_of_sale/static/src/app/screens/product_screen/product_screen.xml

```diff
@@ -12,7 +12,8 @@
                             partner="partner"
                             actionName="constructor.numpadActionName"
                             actionType="'payment'"
-                            onClickMore.bind="displayAllControlPopup" />
+                            onClickMore.bind="displayAllControlPopup"
+                            isActionButtonHighlighted="items"/>
```

Comment:
Putting items there is not the best. Try with refunding a product: when clicking on refund, an order will be created with an orderline of quantity -1. If you click on a product, it will add a line with quantity 1 and the items getter will return 0 so the button will not be highlighted but it should be.
![image](https://github.com/odoo/odoo/assets/118442417/9006ac65-8a4b-412b-831b-4ae817621e7b)

Reply:
Thank you for bringing to my attention that I missed this particular case.